internal/restorer/progressprinter/progressprinter.go

```diff
@@ -0,0 +1,114 @@
+package progressprinter
+
+import (
+	"fmt"
+	"sync"
+	"os"
+	"io"
+	"time"
+)
+
+var progresswriter io.Writer = os.Stdout
+
+// \r = carriage return (cursor to start of line);  \033[K = line deletion (delete from cursor until end of line)
+var lineResetter string = "\r\033[K"
```

Comment:
This is Unix-specific, right?

Reply:
Fixed by simply using \r.
Only crtical moment is when switch unit, solved by adding 3 spaces at end of line